recipes/brpc/all/patches/incubator-brpc-0.9.6-snappy.patch

```diff
@@ -0,0 +1,24 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index cfa0ce66..f244b999 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -164,6 +164,7 @@ include_directories(
+ # PROTOBUF_LIBRARIES. Normally, that should be enabled with
+ # protobuf_MODULE_COMPATIBLE; it appears that the endif for the if (0) in
+ # the protobuf recipe should be moved further up
++find_library(SNAPPY_LIB NAMES snappy)
+ set(DYNAMIC_LIB
+     ${GFLAGS_LIBRARY}
+     ${Protobuf_LIBRARIES}
+@@ -174,9 +175,10 @@ set(DYNAMIC_LIB
+     ${THRIFTNB_LIB}
+     ${OPENSSL_LIBRARIES}
+     ${OPENSSL_CRYPTO_LIBRARY}
++    ${SNAPPY_LIB}
+     dl
+     z)
```

Comment:
Should `zlib/1.2.11` be added as a requirement?
Currently it is maybe added as a transitive requirement, but maybe add it explicitly here?

Reply:
Yeah I think that would be better. I'll do that.